src/css/base.scss

```diff
@@ -169,16 +179,25 @@ fieldset {
 // ----------------------PAST TRIPS-------------------
 .past-trips__wrapper {
   display: flex;
+  overflow-x: scroll;
 }
 
 .past-trips__trip {
+  display: flex;
+  flex-direction: column;
   margin: 10px;
 }
 
 .past-trips__image {
   width: 200px;
 }
 
+.past-trips__trip-wrapper {
```

Comment:
Since you have multiple flexbox properties on elements, maybe this would be a good place to incorporate these into a mixin just to clear up som of your css. But the implementation of all your grid elements looks awesome!